multimedia/whispercpp/Makefile

```diff
@@ -0,0 +1,47 @@
+#
+# Copyright (C) 2024 Dengfeng Liu <liudf0716@gmail.com>
+#
+# This is free software, licensed under the GNU General Public License v3.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=whispercpp
+PKG_VERSION:=v1.5.4
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/ggerganov/whisper.cpp.git
+PKG_SOURCE_VERSION:=$(PKG_VERSION)
+PKG_MIRROR_HASH:=e8a15ad422eb4c11d1c52912e8ce8c422426cb06274d8c2d667f02d9a8859a4e
+
+PKG_MAINTAINER:=Dengfeng Liu <liudf0716@gmail.com>
+PKG_LICENSE:=GPL-3.0-or-later
+PKG_LICENSE_FILES:=COPYING
+
+include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/cmake.mk
+
+define Package/whispercpp
+  SECTION:=multimedia
+  CATEGORY:=Multimedia
+  DEPENDS:=+libstdcpp +ffmpeg
+  TITLE:=Port of OpenAI's Whisper model in C/C++
+  URL:=https://github.com/ggerganov/whisper.cpp
+endef
+
+define Package/whispercpp/description
+  Port of OpenAI's Whisper model in C/C++
+endef
+
+define Package/whispercpp/install
+	$(INSTALL_DIR) $(1)/usr/bin
+	$(INSTALL_DIR) $(1)/usr/lib
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/bin/main $(1)/usr/bin/whispercpp
+	$(INSTALL_DIR) $(1)/usr/share/whispercpp/model
+	$(CP) ./files/ggml-tiny.en.bin $(1)/usr/share/whispercpp/model/
```

Comment:
This file has 74 MB. I dont think that it is intentional. 
Simpy nope.

Reply:
It's the model data that Whisper will use, and it's already the smallest model data.
If a device wants to use it, it must meet the minimum requirements for size.